config/syslog.py

```diff
@@ -642,3 +642,52 @@ def disable_rate_limit_feature(db, service_name, namespace):
         
         if not failed:
             click.echo(f'Disabled syslog rate limit feature for {feature_name}')
+
+
+@syslog.command('level')
+@click.option("-c", "--component",
```

Comment:
@Junchao-Mellanox what is a component?

Reply:
It is the log identifier, `component` is the word used by swssloglevel command. I am trying to follow it.

Reply:
@Junchao-Mellanox OK I understand its used by swssloglevel command. Here what is it used for?

Reply:
it is used as the key of CONFIG DB LOGGER table.

Reply:
@Junchao-Mellanox as a user how do i know which `component` to use? Say I am interested to add debug to Xcvrd in PMON docker.

Reply:
use `swssloglevel -p`, it will print all `component` and their corresponding log level